src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java

```diff
@@ -217,4 +217,68 @@ public boolean packageApp() throws IOException, InterruptedException {
 
         return linkFlags;
     }
+
+    @Override
+    public boolean link() throws IOException, InterruptedException {
+        try {
+            createIconResource();
+            super.link();
+            clearExplorerCache();
+        } catch (IOException ioe) {
+            ioe.printStackTrace();
+            return false;
+        }
+        return true;
+    }
+
+    // During development if user changes the application icon, the same is not reflected immediately in Explorer.
+    // To fix this, a cache clearance of the Windows explorer is required.
+    private void clearExplorerCache() throws IOException, InterruptedException {
+        ProcessRunner clearCache;
+        String osName = System.getProperty("os.name").toLowerCase(Locale.ROOT);
+        // For Windows version < 10, use `ie4uinit.exe -ClearIconCache`
+        if (osName.equals("windows 10") || osName.equals("windows 11")) {
+            clearCache = new ProcessRunner("ie4uinit", "-show");
```

Comment:
You can do `ProcessRunner clearCache = new ProcessRunner("ie4uinit");`
And then, with an `if` or a ternary expression:
`clearCache.addArg(10+? "-show" : "-ClearIconCache");`